packages/opentelemetry-sdk-trace-base/src/export/ConsoleSpanExporter.ts

```diff
@@ -65,6 +65,7 @@ export class ConsoleSpanExporter implements SpanExporter {
       resource: {
         attributes: span.resource.attributes,
       },
+      instrumentationLibrary: span.instrumentationLibrary,
```

Comment:
Aside: Do you think we'll ever change the property to `span.instrumentationScope` given that the `InstrumentationLibrary` type is deprecated (https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-core/src/common/types.ts#L42-L63) ?

Reply:
I think we should emit it with the name `instrumentationScope`. Updated the PR Thanks 👍